src/js/containers/generateFiles/GenerateFilesContainer.jsx

```diff
@@ -401,6 +418,14 @@ class GenerateFilesContainer extends React.Component {
 					show: true,
 					url: fileData.url
 				};
+				if(fileData.url === '' || fileData.url === '#' || fileData.url == null){
```

Comment:
you might be able to use Lodash here to simplify the check instead of doing 3 conditionals combined with `||`: http://stackoverflow.com/a/35823806